extensions/blocks/donations/icon.js

```diff
@@ -1,20 +1,6 @@
 /**
  * External dependencies
  */
-import { SVG, Path, Circle } from '@wordpress/components';
+import Gridicon from '../../../_inc/client/components/gridicon';
```

Comment:
This isn't going to work on WordPress.com, where that directory does not exist.

What would you think about doing something like this instead?
https://github.com/Automattic/jetpack/blob/409f77474ea16cf16691e2ee2374655d65a74e9c/extensions/shared/components/upgrade-nudge/index.jsx#L4

Reply:
Makes sense! I had a feeling that might turn out not to be the ideal import statement. Thanks so much for the pointer!